TODO.md

```diff
@@ -0,0 +1,32 @@
+# CNCF Hugo Starter Checklist
```

Comment:
#2: This could always change, but most projects are coming in through sandbox these days. I think you can expect the majority of projects to come in with their own design, call it 90%. There may be 10% that come in with just a GH repo, but that's not common.